components/03-modules/side-menu/side-menu.config.js

```diff
@@ -0,0 +1,67 @@
+module.exports = {
+  title: 'Side menu',
+  context: {
+    list: [
+      {
+        text: 'Furniture',
+        class: 'mega-menu__item mega-menu__item--level0 mega-menu__item--parent',
+        nestedList: [
+          {
+            class: 'mega-menu__inner-item mega-menu__inner-item--level2',
+            text: 'Lorem ipsum',
+            elementLink: '#',
```

Comment:
`elementLink` it's not used, to add `href="#"` you should use `itemAttributes` from the `dropdown-list` config, please check every element with `a href` here

Reply:
it's legacy code, I think we can remove whole file. Is it ok?

Reply:
Indeed, we don't need the config file for this... I thought that the config is needed to display the component 👯  but as we see it's working well, so IMO it's ok :)